### PR TITLE
you now always face the direction you're crawling

### PR DESCRIPTION
**Pull request by Couls**
## About The Pull Request

When crawling your head will always face the direction you're crawling in
## Why It's Good For The Game

More immersive, less people crawling with their legs first
ezgif com-video-to-gif

## Changelog
:cl:
tweak: people now crawl in the direction their heads are facing
/:cl:

### CODE REVIEW

**Comment by Twaticus** [comment]
youve never had to crawl feet first before? i think seeing a horizontal spaceman spam flip flop their direction is going to be more immersion breaking than what we currently have

**Comment by Couls** [comment]
A bit harder to do because of crawling slowdown and it doesn't actually change direction unless you move so you can't do like the ctrl + a, ctrl + d spam to look silly on the floor

**Comment by cacogen** [comment]
While we're bikeshedding, you should only ever be face down while crawling

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -585,6 +585,14 @@
...
 
 /mob/living/Move(atom/newloc, direct)
+	if(lying && direct & EAST)
+		lying = 90
+		update_transform()
+		lying_prev = lying
+	else if(lying && direct & WEST)
+		lying = 270
+		update_transform()
+		lying_prev = lying
```
Comment (nemvar) [inline code/modules/mob/living/living.dm]: Don't check for lying twice and get rid of the duplicate code.
```
	if(lying) 
		if(direct & EAST)
			lying = 90
		if(direct & EAST)
			lying = 270
		update_transform()
		lying_prev = lying
```

**Comment by 81Denton** [comment]
Travis failure is unrelated to this PR, this is good to go.

**Pull request merged**